Example/Podfile

```
@@ -1,29 +1,32 @@
+source 'https://github.com/bitmovin/cocoapod-specs.git'
+source 'https://github.com/CocoaPods/Specs.git'
+
 def shared_pods
   pod 'BitmovinConvivaAnalytics', path: '../'
-  pod 'BitmovinPlayer', git: 'https://github.com/bitmovin/bitmovin-player-ios-sdk-cocoapod.git', tag: '2.64.0'
+  pod 'BitmovinPlayer', '3.6.1-rc.2'
```

Let's use the stable version instead of the RC as it's available by now.

```suggestion
  pod 'BitmovinPlayer', '3.6.1'
```